scripts/host-only/wkdev-sdk-bakery

```diff
@@ -19,6 +19,7 @@ argsparse_use_option =verbose     "Increase verbosity of this script"
 
 argsparse_use_option =name:       "Name of container image" mandatory default:wkdev-sdk
 argsparse_use_option =mode:       "Operation mode: 'build', 'deploy', or 'export'" mandatory
+argsparse_use_option =platforms:  "Build container for given set of platforms" mandatory default:"linux/arm/v7,linux/amd64"
```

Comment:
I'd think the default should just be current architecture, with CI doing more.

Reply:
Correct.